templates/download/server/manual.html

```diff
@@ -166,9 +166,133 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
+          <hr />
+          <p class="u-responsive-realign">
+            <a class="p-button--positive"
+               href="/download/server/thank-you?version={{ releases.latest.full_version }}&amp;architecture=amd64&amp;lts=true"
+               onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Server download', 'eventAction' : 'Select option', 'eventLabel' : '{{ releases.latest.full_version }}', 'eventValue' : undefined });">Download {{ releases.latest.full_version }}
+            </a>
+            {% if releases.latest.server_iso_size %}
+              <span class="u-text--muted">{{ releases.latest.server_iso_size }}</span>
+            {% endif %}
+          </p>
+        </div>
+
+        <div class="js-tab-container">
+          <nav class="p-tabs" id="second-child-tab">
+            <ul class="p-tabs__list js-tabbed-content u-no-margin--bottom"
+                role="tablist"
+                data-maintain-hash="true">
+              <li class="p-tabs__item">
+                <a class="p-tabs__link"
+                   role="tab"
+                   aria-selected="true"
+                   id="release-notes-tab-latest"
+                   href="#release-notes-latest"
+                   aria-controls="release-notes-latest">What's new</a>
+              </li>
+              <li class="p-tabs__item">
+                <a class="p-tabs__link"
+                   role="tab"
+                   aria-selected="false"
+                   id="system-requirements-tab-latest"
+                   href="#system-requirements-latest"
+                   aria-controls="system-requirements-latest"
+                   tabindex="-1">System requirements</a>
+              </li>
+              <li class="p-tabs__item">
+                <a class="p-tabs__link"
+                   role="tab"
+                   aria-selected="false"
+                   id="how-to-install-tab-latest"
+                   href="#how-to-install-latest"
+                   aria-controls="how-to-install-latest"
+                   tabindex="-1">How to install</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+
+        <div class="p-tabs__content p-section"
+             id="release-notes-latest"
+             role="tabpanel"
+             aria-labelledby="release-notes-tab-latest">
+          <ul class="p-list--divided">
+            <li class="p-list__item is-ticked">Linux 6.11 kernel</li>
+            <li class="p-list__item is-ticked">Addition of kdump-tools for automatic kernel crash dumps</li>
+            <li class="p-list__item is-ticked">Inclusion of Valkey, a high performance key/value data store.</li>
+            <li class="p-list__item is-ticked">
+              Toolchain updates now include versioned Rust packages and TCK certified OpenJDK 21 and 17
+            </li>
+          </ul>
+          <hr class="p-rule" />
+          <ul class="p-inline-list" style="padding-top: 0.375rem;">
```

Comment:
wrapping the `ul` with a `p` seems to be working weirdly so i just added an inline style instead. It could possibly be a bug with vanilla..